facebook/OpenBIC
# fby3.5: rf: Support IPMB

Summary:
- Support IPMB handler.

Dependency: #299 

Test plan:
- Build code: Pass
- Boot successful: Pass
- Check IPMB is workable: Pass

Log:
1. Check RF BIC can boot successful.
[BIC console]
0)I?** Booting Zephyr OS build v00.01.04-5-g44d2dfad91aa  ***
Hello, welcome to yv35 Rainbow Falls 2022.20.e3
Function pal_fix_full_sdr_table is not implemented
The SDR number is equal to 0
ipmi_init

uart:~$

2. Check IPMB is workable via IPMI command.
[BMC console]
root@bmc-oob:~# fw-util slot2 --version 1ou_bic
1OU Bridge-IC Version: oby35-rf-v2022.20.e3
